recipes/asio/all/conanfile.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import os
+from conans import ConanFile, tools
+
+
+class Asio(ConanFile):
+    name = "asio"
+    
+    url = "https://github.com/chriskohlhoff/asio"
+    homepage = "http://think-async.com/Asio"
+    description = "Asio is a cross-platform C++ library for network and low-level I/O"
+    topics = ("conan", "asio", "network", "io", "low-level")
+    license = "BSL-1.0"
+
+    settings = "os", "arch", "compiler", "build_type"
```

Comment:
```suggestion
```

Reply:
The `self.info.header_only()` already does the job... I'll remove them because it is the way we are doing it, but I'd like to revisit this at some moment